<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiLayerItem</class>
 <widget class="QWidget" name="guiLayerItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{

/*
    border-top: 1px solid rgb(100,100,100);
    border-left: 1px solid rgb(100,100,100);
    border-right: 1px solid rgb(90,90,90);
    border-bottom: 1px solid rgb(75,75,75);

background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(45, 62, 109, 255), stop:1 rgba(65, 97, 145, 255));
	border: 1px solid grey;
	border-radius: 2px;
*/
	border: 0px;	

	color: white;

	padding: 4px;
	margin: 2px;
}



QPushButton{
	color: rgba(255,255,255,200);
	background-color: rgb(80,80,80);
	border: 1px solid rgba(255,255,255,100);
	border-radius: 2px;
}

QPushButton::hover{
	color: rgba(255,255,255,255);
	background-color: rgb(100,100,100);
}

QToolButton{
	color: rgba(255,255,255,200);
	background-color: rgb(80,80,80);
	border: 1px solid rgba(255,255,255,100);
	border-radius: 2px;
}

QToolButton::hover{
	color: rgba(255,255,255,255);
	background-color: rgb(100,100,100);
}

/********************************************************
/* Sliders
/*******************************************************/

QSlider{
	background: rgba(0,0,0,0);
}

QSlider::groove:horizontal {
     border: 1px solid #999999;
	border-radius: 5px;
     height: 12px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgba(75,75,75, 255), stop:1 rgba(59,59,59, 255));
     margin: 2px 0;
 }

QSlider::handle:horizontal {
     border: 0px;
	background-color: rgb(171, 171, 171);
     width: 12px;
     margin: 0px 0 0px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
	border-radius: 5px;
 }

QSlider::handle:horizontal:hover {
     border: 0px;
	background-color: rgb(200, 200, 200);
     width: 12px;
     margin: 0px 0 0px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
	border-radius: 5px;
 }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,0">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click for layer menu</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="gainSlider">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>127</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_solo">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton::Checked {
	
	background-color: rgb(154, 132, 6);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>S</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_mute">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton::Checked {
	
	background-color: rgb(173, 0, 0);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>M</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_bus">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionEdit_Filter">
   <property name="text">
    <string>Edit MIDI Filter...</string>
   </property>
  </action>
  <action name="actionRemove_Layer">
   <property name="text">
    <string>Remove Layer</string>
   </property>
  </action>
  <action name="actionReload_Layer">
   <property name="text">
    <string>Reload Layer</string>
   </property>
   <property name="toolTip">
    <string>Reload Layer</string>
   </property>
  </action>
  <action name="actionOpen_in_File_Manager">
   <property name="text">
    <string>Open in File Manager</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
